contracts/mixnet/src/mixnodes/transactions.rs

```diff
@@ -248,7 +277,19 @@ pub(crate) fn try_update_mixnode_config(
             .ok_or(ContractError::NoBondFound)
     })?;
 
-    Ok(Response::new())
+    let mut response = Response::new();
+
+    if let Some(proxy) = proxy {
+        // Returns one_unym proxy had to send in order to execute the contract to contract transaction, this is potentially leaky as anyone can say that they're a proxy,
+        // and they could potentially leak 1 unym per transaction, altough I'm pretty sure transaction fees make that silly.
+        let return_one_unymt = BankMsg::Send {
+            to_address: proxy.as_str().to_string(),
+            amount: vec![one_unym()],
+        };
+        response = response.add_message(return_one_unymt);
```

Comment:
The variable name and response message (and fwiw the comment too) aren't valid for all networks